lib/utility.js

```diff
@@ -42,10 +42,8 @@ async function loadLocalConf() {
 }
 
 export async function loadData(id) {
-	/* Check for config override file. */
-	const lconf = await loadLocalConf()
-	const path = lconf?.data_paths?.[id] ??
-		('../data/' + id + '.js')
+	const path = globalThis.VITEPRESS_CONFIG.userConfig.themeConfig.dovecot?.data_paths?.[id]
+		?? ('../data/' + id + '.js')
```

Comment:
Can this not use the `dovecotSetting()` macro?
i.e. `dovecotSetting('data_paths')?.[id]` or am I holding this wrong?

Reply:
You can't, because dovecot_setting() would require the 'data_paths' setting to always exist.  All the data_path settings are optional, including the top-level object.